### PR TITLE
fix to show product name on autocomplete options

### PR DESCRIPTION
**Pull request by rayssamfreitas**
#### What does this PR do? \*

This PR modifies the list of autocomplete options to display the product name rather than the product SKU name.

#### How to test it? \*

In 'One by One' mode, type the name of a product, and a list of related products will appear instead of a list of related SKUs.

### CODE REVIEW

**Comment by rayssamfreitas** [comment]
Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖

Please select which version do you want to release:

[x] Patch (backwards-compatible bug fixes)
[ ] Minor (backwards-compatible functionality)
[ ] Major (incompatible API changes)

And then you just need to merge your PR when you are ready! There is no need to create a release commit/tag.

[ ] No thanks, I would rather do it manually 😞

**Comment by rayssamfreitas** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [x]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉

**Comment by rayssamfreitas** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [x]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

File: react/components/QuickOrderAutocomplete.tsx [hunk 1]
```diff
@@ -116,17 +114,17 @@ const QuickOrderAutocomplete: FunctionComponent<
...
-              slug: item.linkText,
-              thumb: getImageSrc(item.items[0].images[0].imageUrl),
-            }
-          }),
+        .filter((item: any) => {
+          return !!item.items[0].images[0].imageUrl
+        })
+        .map((item: any) => {
+          return {
+            value: item.productName,
```
Comment (ataideverton) [inline react/components/QuickOrderAutocomplete.tsx:122]: Is there any need to change the value too? It's not just the label? Having the value as a long string with special characters might not be the best thing

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```
Comment (ataideverton) [inline .vscode/settings.json:4]: Why did you change this?

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [3.15.3] - 2024-06-10
 
+### Changed
+
+- Adjust autocomplete list to show product name rather then product sku name
```
Comment (ataideverton) [inline CHANGELOG.md:14]:
```suggestion
- Adjust autocomplete list to show product name rather than just the sku name
```

**Review by ataideverton — CHANGES_REQUESTED** [review]

File: react/components/QuickOrderAutocomplete.tsx [hunk 4]
```diff
@@ -116,17 +114,17 @@ const QuickOrderAutocomplete: FunctionComponent<
...
-              slug: item.linkText,
-              thumb: getImageSrc(item.items[0].images[0].imageUrl),
-            }
-          }),
+        .filter((item: any) => {
+          return !!item.items[0].images[0].imageUrl
+        })
+        .map((item: any) => {
+          return {
+            value: item.productId,
```
Comment (ataideverton) [inline react/components/QuickOrderAutocomplete.tsx:122]: Might be better to just use the skuId, like it was before, to avoid any possibility of the same product returning twice and duplicating values.

No need to change the value, just the label.

**Review by ataideverton — CHANGES_REQUESTED** [review]

**Review by ataideverton — APPROVED** [review]
